### DISCUSSION TITLE
sir,Fix tsync error for Project-Kaleidoscope-sunflowerleaf

### DISCUSSION THREAD

**Discussion by mitmac29** [ROM-builders/temporary#19695] [General]
Even I tried clean build , still there is Tsync error

Failed builds : 
https://cirrus-ci.com/build/5187613552803840

https://cirrus-ci.com/task/5729805862174720

Thanks sir !!!

**Comment by Apon77** [comment]
It's not fixable.and rom is outdated.so leave

**Comment by hipexscape** [comment]
> Even I tried clean build , still there is Tsync error
> 
> Failed builds : https://cirrus-ci.com/build/5187613552803840
> 
> https://cirrus-ci.com/task/5729805862174720
> 
> Thanks sir !!!

It can't be fixed
